.github/workflows/create-release.yml

```diff
@@ -39,17 +39,24 @@ jobs:
       - run: |
           echo "Creating version ${{ steps.collect-data.outputs.version }} from ${{ steps.collect-data.outputs.branch }} with artifacts from build ${{ steps.collect-data.outputs.runId }} "
           
+  owasp-scanner:
+    needs: [ setup ]
+    uses: liquibase/build-logic/.github/workflows/owasp-scanner.yml@DAT-16650
```

Comment:
Just remember to remove the DAT reference here befopre merging.

Reply:
done